src/main/java/model/Cars.java

```diff
@@ -0,0 +1,62 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.toList;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if (cars.isEmpty()) {
+            throw new IllegalArgumentException("레이싱 게임에 참가할 자동차를 1대 이상 입력해주세요");
+        }
+
+        this.cars = cars;
+    }
+
+    public Cars move() {
+        List<Car> records = new ArrayList<>();
+        for (Car car : cars) {
+            Car moved = car.move();
+            records.add(moved);
+        }
+
+        return new Cars(records);
+    }
+
+    public List<String> winnerNames() {
```

Comment:
```suggestion
    public List<Car> winners() {
```
우승자들을 제공하고 어떻게 출력하는지는 view 영역에서 판단하면 될 것 같아요
요구사항이 변경되어 나이까지 출력하라고 하면 도메인의 변경이 발생되고 물론 메시지를 수신받는 쪽도 영향이 가게됩니다.